lib/tasks/stimulus_tasks.rake

```diff
@@ -46,16 +55,16 @@ namespace :stimulus do
 
   namespace :manifest do
     desc "Show the current Stimulus manifest (all installed controllers)"
+
     task :display do
-      puts Stimulus::Manifest.generate_from(Rails.root.join("app/javascript/controllers"))
+      Stimulus::Manifest.generate_from(Stimulus::Tasks.controllers_path)
```

Comment:
Why remove the puts here? That would remove the actual display of the Manifest.

Reply:
Gladly.  It was nice when I was debugging, but I'm usually beaten with a stick for leaving `puts` in code. 😅

Reply:
puts for debugging is definitely not something you want to ship, but this puts is in service of what the task does: Display the manifest to the user on the terminal.

Reply:
@dhh What are your thoughts adding a puts to `stimulus:manifest:update` to avoid surprise when `js_root` is not default?

```
desc "Update the Stimulus manifest (will overwrite controllers/index.js)"
    task :update do
      puts "Updating stimulus manifest in #{Stimulus::Tasks.controllers_path.relative_path_from(Rails.root)}"
      manifest = Stimulus::Manifest.generate_from(Stimulus::Tasks.controllers_path)
       ....
```